.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -379,10 +389,17 @@
         </ItemGroup>
 
         <!-- iOS, but not Catalyst -->
+        <ItemGroup Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'">
+            <!-- Add a Link attribute in the form of '<filename>_<inputshash>.<extension>' -->
+            <_MauiSplashScreenWithLink Include="@(_MauiSplashScreenWithHashes->HasMetadata('Link'))" />
+            <_MauiSplashScreenWithoutLink Include="@(_MauiSplashScreenWithHashes)" Exclude="@(_MauiSplashScreenWithLink)" />
+            <_MauiSplashScreenWithHashesInFilename Include="@(_MauiSplashScreenWithoutLink)" Link="%(Filename)_$(_MauiSplashInputsFileHash)%(Extension)" />
+            <_MauiSplashScreenWithHashesInFilename Include="@(_MauiSplashScreenWithLink)" Link="$([System.IO.Path]::GetFilenameWithoutExtension('%(Link)'))_$(_MauiSplashInputsFileHash)$([System.IO.Path]::GetExtension('%(Link)'))" />
```

Comment:
Only suggestion is these item groups look tricky. When the XML starts looking ugly, I sometimes move logic like this to C# instead. Most folks would look at this and have no idea about it. 👀

Reply:
Ah thanks. Didn't want to write C# here, but you gave me an idea. The new way is much simpler...ish